atomix/cluster/src/main/java/io/atomix/raft/RaftServer.java

```diff
@@ -22,20 +22,14 @@
 
 import io.atomix.cluster.ClusterMembershipService;
 import io.atomix.cluster.MemberId;
-import io.atomix.primitive.PrimitiveTypeRegistry;
-import io.atomix.primitive.operation.OperationType;
-import io.atomix.primitive.service.PrimitiveService;
 import io.atomix.raft.cluster.RaftCluster;
 import io.atomix.raft.cluster.RaftMember;
 import io.atomix.raft.impl.DefaultRaftServer;
 import io.atomix.raft.impl.RaftContext;
-import io.atomix.raft.impl.RaftServiceManager;
+import io.atomix.raft.impl.zeebe.ZeebeRaftStateMachine;
```

Comment:
Nit: I'd be fine collapsing this and the interface type, or at least renaming to something more descriptive than Zeebe :sweat_smile: